src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/DeciderActionPriorityReader.java

```diff
@@ -45,8 +45,7 @@ public static void updateDeciderActionPriorityOrder() {
         try {
             deciderActionPriorityOrder = mapper.readValue(new File(configFilePath), HashMap.class);
         } catch (Exception e) {
-            LOG.error("Could not read the Decider Config File");
-            e.printStackTrace();
+            LOG.error("Could not read the Decider Config File: " + configFilePath);
```

Comment:
Log.error("Could not read the Decider Config File: {}", configFilePath, e)

Reply:
This file has been removed in the latest update to master.